code/game/objects/structures/extinguisher.dm

```diff
@@ -141,16 +141,22 @@ MAPPING_DIRECTIONAL_HELPERS(/obj/structure/extinguisher_cabinet, 29)
 		update_appearance()
 
 /obj/structure/extinguisher_cabinet/update_icon_state()
-	if(!opened)
-		icon_state = "extinguisher_closed"
-		return ..()
+	icon_state = "extinguisher"
+
 	if(!stored_extinguisher)
```

Comment:
```suggestion
	if(isnull(stored_extinguisher))
```

more explicit since this is a typed var and not a boolean. is also like 0.001x faster (but that's not the point)